index.js

```diff
@@ -37,6 +37,17 @@ module.exports = (milliseconds, options = {}) => {
 		}
 	}
 
+	if (
+		milliseconds >= (1000 * 60) - 50 &&
```

Comment:
`50` should be a constant because it's not clear what it's for.

Reply:
Should be clear now